### DISCUSSION TITLE
apt-key is Depreciated (Docs)

### DISCUSSION THREAD

**Discussion by mtkappen** [requarks/wiki#5391] [Error / Bug Report] [answered]
In the Wiki docs for Ubuntu installation, to register the docker package registry it requires the usage of apt-key. Apt-key is depreciated and the docs will need to be adjusted. I would do it myself but I don't know what the new way is.

I don't know if this is supposed to go here, but the wiki-docs repository doesn't have its own discussions board to my knowledge.

**Comment by NGPixel** [comment]
You can submit a PR with the fix on the wiki-docs repository.

**Reply by mtkappen** [reply]
No, you misunderstand. I don't know what the solution to the problem is. I just know it is a problem. I was asking if this was the proper place to submit what I would consider a bug for the docs as you cannot install wiki.js on Ubuntu following the currently listed instructions.

I just wanted you to be aware of the problem. My experience is limited but if I manage to figure it out before someone else, then I will submit a pull request.

**Comment by NGPixel** [comment, marked as answer]
Docs have been updated.
